checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -299,6 +301,12 @@ protected AnnotatedTypeFormatter createAnnotatedTypeFormatter() {
                 super.methodFromUse(tree);
         AnnotatedExecutableType method = mfuPair.first;
 
+        if (method.getReturnType().getKind() == TypeKind.WILDCARD
```

Comment:
Can we make this a deep check, to cover if the uninferred wildcard occurs as an array component type or as a type argument? Those cases also happen.